gage_outages/outage_maps.R

```diff
@@ -228,18 +251,20 @@ qpf <- sf::st_intersection(qpf, sf::st_buffer(sf::st_as_sf(conus), 0))
 ################################
 # Plot it up
 ################################
-
+states.out_sf <- sf::st_as_sf(states.out)
 # Shapes by site type:
 gsMap <- ggplot() +
-  geom_polygon(aes(x = long, y = lat, group = group),
-               data = states.out, fill = "grey90",
-               alpha = 0.9, color = "grey") +
+  # geom_polygon(aes(x = long, y = lat, group = group),
```

Comment:
Trying to get Colorado to plot underneath everything else. No dice.

Reply:
so you prefer the geom_sf?

Reply:
I thought maybe using sf would fix the Colorado issue.